servers/parser-graphql-wrapper/schema.graphql

```diff
@@ -480,4 +480,9 @@ type ReaderInterstitial {
 
 type ItemNotFound {
   message: String 
-}
\ No newline at end of file
+}
+
+type PocketShare @key(fields: "targetUrl") {
+  targetUrl: Url!
+  preview: ItemSummary
```

Comment:
This federation needs to be enabled on the parser side yea?

Reply:
Yep, it's on the parser subgraph